feature-utils/poly-analysis/src/context/poly-analysis.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useContext, useEffect, useState } from "react";
+
+export const PolyAnalysisContext = createContext();
+
+export const PolyAnalysisProvider = ({
+    children,
+    parentContext,
+    subAnalyses,
+    analyzeFile,
+}) => {
+    const { account, files } = useContext(parentContext);
```

Comment:
This is what I mean in my rant below. We're actually using account and files (as a matter of fact, only the first file) as props, not really as a dynamically updated context, or as context consumers.